code/game/gamemodes/shadowling/shadowling_abilities.dm

```diff
@@ -56,6 +56,18 @@
 		for(var/obj/item/device/pda/P in orange(5, usr))
 			P.fon = 0
 			P.SetLuminosity(0) //failsafe
+		for(var/obj/effect/glowshroom/G in orange(3, usr)) //Smaller radius
+			qdel(G)
+		for(var/mob/living/carbon/human/H in T.contents)
+			for(var/obj/item/device/flashlight/F in H)
+				if(is_type_in_list(F, blacklisted_lights))
+					F.visible_message("<span class='danger'>[F] goes slightly dim for a moment.</span>")
+					return
+				F.on = 0
+				F.visible_message("<span class='danger'>[F] gutters and falls dark.</span>")
+				F.update_brightness()
+			H.luminosity = 0 //This is required with the object-based lighting
+			H.update_brightness()
```

Comment:
Also, this just causes PDA lights to stop working until you move them around in your inventory. Might want to fix that.